deploy/template.yaml

```diff
@@ -2543,6 +2543,36 @@ Resources:
         SSEType: KMS
         KMSMasterKeyId: !GetAtt DynamoDBKmsKey.Arn
 
+  SessionCredentialsTable:
```

Comment:
Should we be adding a [TTL specification](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-dynamodb-table-timetolivespecification.html) to the table as well?

Reply:
Oh snap. That's what you get for copying a table without one 😓